DesignRules.md

```diff
@@ -446,6 +446,39 @@ Resources are often interconnected by relationships. Relationships can be modell
   <p></p>
 </div>
 
+## Conditional Modules
+
+As described in <a href="#extensions">the Extensions paragraph</a> the NL API Strategy is composed of a set of modules that are extentions on <a href="#the-core-set-of-design-rules">the Core set of Design Rules</a>. The modules are optional or conditional. Optional rules can be found in the infographic. All conditional modules are referenced in this document and the 'conditional usage' is formalized as a unique designrule.
+
+<span id="api-tls"></span>
+<div class="rule" id="/core/conditional/transport-security">
+  <p class="rulelab"><b>/core/conditional/transport-security</b>: Apply transport security conform the module</p>
+   <dl>
+      <dt>Statement</dt>
+      <dd>
+         When handling government data the transport security module MUST be applied.
+      </dd>
+      <dt>Rationale</dt>
+      <dd>
+         The transport security module as described and published on <a href="https://github.com/Geonovum/KP-APIs">Github</a> formalizes three simple rules to apply to API's:
```

Comment:
```suggestion
         The transport security module as described and published on <a href="https://github.com/Geonovum/KP-APIs">Github</a> formalizes three rules to apply to APIs:
```
De regel "No sensitive information in URIs" had was discussie, dus wellicht "simple" weglaten.